### PR TITLE
[DOC] Update Readme

### PR DESCRIPTION
**Pull request by xf-**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,15 @@
 - Node filter is implemented
 - Zoom level if you click a node (`nodeZoom`) - Zoom level 22 available, but it is to close for a click
 - Formatted Code
-- Translation support - https://crowdin.com/project/meshviewer
+- Translation support - https://crowdin.com/project/meshviewer - Contact us for new languages
```
Comment (genofire) [inline README.md:11]: wo kontaktieren?

**Review by genofire — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xf-) [reply README.md:11]: We use github and we have a support section in Readme.

Comment (genofire) [reply README.md:11]: Create a Issue for new languages?

Comment (xf-) [reply README.md:11]: for exmaple or ask in irc

Comment (genofire) [inline README.md]: - Create a issue for new languages?

**Review by genofire — APPROVED** [review]

**Pull request merged**